examples/lvgldemo/lvgldemo.c

```diff
@@ -89,6 +91,25 @@ void lv_demo_widgets(void);
  * Private Functions
  ****************************************************************************/
 
+/****************************************************************************
+ * Name: monitor_cb
+ *
+ * Description:
+ *   Monitoring callback from lvgl every time the screen is flushed.
+ *
+ ****************************************************************************/
+
+static void monitor_cb(lv_disp_drv_t * disp_drv, uint32_t time, uint32_t px)
+{
+  ginfo("%" PRIu32 " px refreshed in %" PRIu32 " ms\n", px, time);
+}
+
+/****************************************************************************
+ * Private Data
+ ****************************************************************************/
+
+lv_color_t buf[DISPLAY_BUFFER_SIZE];
```

Comment:
```suggestion
static lv_color_t buf[DISPLAY_BUFFER_SIZE];
```
This buffer does not need to have external linkage.